doc/api/next_api_changes/deprecations.rst

```diff
@@ -67,3 +67,9 @@ This attribute is unused and deprecated.
 ``widgets.TextBox.params_to_disable``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 This attribute is deprecated.
+
+De-deprecate \*min, \*max keyword arguments to ``set_x/y/zlim_3d()``
```

Comment:
Can we change this to "Undo deprecation of ..." ?  "De-deprecate" gave me a headache....